<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.361">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>antibioticsbook - 4&nbsp; Antibiotic testing the in the laboratory</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./resistance1.html" rel="next">
<link href="./history.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./antibiotictesting.html"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Antibiotic testing the in the laboratory</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">antibioticsbook</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Preface</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./intro.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Introduction</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./history.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">History of Antibiotic Development</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./antibiotictesting.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Antibiotic testing the in the laboratory</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resistance1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Antibiotic resistance, part 1</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resistance2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Antibiotic resistance, part 2</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./eskape.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Multidrug resistance:The ESKAPE pathogens</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./discovery1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Antibiotioc discovery, part 1</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./discovery2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Antibiotic discovery, part 2</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preclinical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Preclinical development of antibiotic candidates</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./toxicities.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Evaluation of antibiotic toxicities</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./susceptibility.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Clinical application of antimicrobial susceptibility testing</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./trial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Clinical trial design</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./economics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Economics of antibiotic discovery</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./diagnostictests.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Diagnostic testing and the micrbiology laboratory</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./stewardship.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Antimicrobial stewardship</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./lmics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Antibiotic access in low-middle income countries</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./mipd.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Model-informed precision dosing and therapeutic drug monitoring</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./summary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Summary</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">4.1</span> Introduction</a></li>
  <li><a href="#the-concept-of-antimicrobial-spectrum" id="toc-the-concept-of-antimicrobial-spectrum" class="nav-link" data-scroll-target="#the-concept-of-antimicrobial-spectrum"><span class="header-section-number">4.2</span> The concept of antimicrobial spectrum</a>
  <ul class="collapse">
  <li><a href="#the-gram-stain-test" id="toc-the-gram-stain-test" class="nav-link" data-scroll-target="#the-gram-stain-test"><span class="header-section-number">4.2.1</span> The Gram-stain test</a></li>
  <li><a href="#why-not-prescribe-the-broadest-spectrum-antimicrobial-for-all-infections" id="toc-why-not-prescribe-the-broadest-spectrum-antimicrobial-for-all-infections" class="nav-link" data-scroll-target="#why-not-prescribe-the-broadest-spectrum-antimicrobial-for-all-infections"><span class="header-section-number">4.2.2</span> Why not prescribe the broadest-spectrum antimicrobial for all infections?</a></li>
  </ul></li>
  <li><a href="#bactericidal-versus-bacteriostatic-activity" id="toc-bactericidal-versus-bacteriostatic-activity" class="nav-link" data-scroll-target="#bactericidal-versus-bacteriostatic-activity"><span class="header-section-number">4.3</span> Bactericidal versus bacteriostatic activity</a></li>
  <li><a href="#antimicrobial-susceptibility-testing--the-minimum-inhibitory-concentration-mic" id="toc-antimicrobial-susceptibility-testing--the-minimum-inhibitory-concentration-mic" class="nav-link" data-scroll-target="#antimicrobial-susceptibility-testing--the-minimum-inhibitory-concentration-mic"><span class="header-section-number">4.4</span> Antimicrobial susceptibility testing- The minimum inhibitory concentration (MIC)</a>
  <ul class="collapse">
  <li><a href="#broth-based-mic-testing-methods." id="toc-broth-based-mic-testing-methods." class="nav-link" data-scroll-target="#broth-based-mic-testing-methods."><span class="header-section-number">4.4.1</span> Broth-based MIC testing methods.</a></li>
  </ul></li>
  <li><a href="#automated-antimicrobial-susceptibility-testing" id="toc-automated-antimicrobial-susceptibility-testing" class="nav-link" data-scroll-target="#automated-antimicrobial-susceptibility-testing"><span class="header-section-number">4.5</span> Automated antimicrobial susceptibility testing</a></li>
  <li><a href="#rapid-phenotypic-antimicrobial-susceptibility-testing" id="toc-rapid-phenotypic-antimicrobial-susceptibility-testing" class="nav-link" data-scroll-target="#rapid-phenotypic-antimicrobial-susceptibility-testing"><span class="header-section-number">4.6</span> Rapid phenotypic antimicrobial susceptibility testing</a>
  <ul class="collapse">
  <li><a href="#genetic-detection-for-rapid-detection-of-resistance-markers" id="toc-genetic-detection-for-rapid-detection-of-resistance-markers" class="nav-link" data-scroll-target="#genetic-detection-for-rapid-detection-of-resistance-markers"><span class="header-section-number">4.6.1</span> Genetic detection for rapid detection of resistance markers</a></li>
  </ul></li>
  <li><a href="#technical-limitations-of-phenpotypic-mic-testing" id="toc-technical-limitations-of-phenpotypic-mic-testing" class="nav-link" data-scroll-target="#technical-limitations-of-phenpotypic-mic-testing"><span class="header-section-number">4.7</span> Technical limitations of phenpotypic MIC testing</a></li>
  <li><a href="#how-are-mic-results-interpreted" id="toc-how-are-mic-results-interpreted" class="nav-link" data-scroll-target="#how-are-mic-results-interpreted"><span class="header-section-number">4.8</span> How are MIC results interpreted?</a></li>
  <li><a href="#rapid-genotypic-antimicrobial-susceptibility-testing" id="toc-rapid-genotypic-antimicrobial-susceptibility-testing" class="nav-link" data-scroll-target="#rapid-genotypic-antimicrobial-susceptibility-testing"><span class="header-section-number">4.9</span> Rapid genotypic antimicrobial susceptibility testing</a></li>
  <li><a href="#key-concepts-for-review" id="toc-key-concepts-for-review" class="nav-link" data-scroll-target="#key-concepts-for-review"><span class="header-section-number">4.10</span> Key concepts for review</a></li>
  <li><a href="#lecture-slides" id="toc-lecture-slides" class="nav-link" data-scroll-target="#lecture-slides"><span class="header-section-number">4.11</span> Lecture slides</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">4.12</span> References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Antibiotic testing the in the laboratory</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>Learning objectives</p>
<ul>
<li><p>ddd</p></li>
<li><p>ddd</p></li>
<li><p>ddd</p></li>
</ul>
</div>
</div>
<section id="introduction" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="introduction"><span class="header-section-number">4.1</span> Introduction</h2>
<p>Antimicrobial susceptibility testing is a crucial process in the field of microbiology and infectious diseases. It involves determining the effectiveness of various antibiotics or antimicrobial agents against specific bacteria or other microorganisms. By conducting this testing, healthcare professionals can make informed decisions about which antibiotics or treatments will be most effective in combating infections. This testing is also performed routinely on large collections of isolates to detect epidemiological trends of resistance (surveillance).</p>
</section>
<section id="the-concept-of-antimicrobial-spectrum" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="the-concept-of-antimicrobial-spectrum"><span class="header-section-number">4.2</span> The concept of antimicrobial spectrum</h2>
<p>The activity spectrum of an antibacterial drug is related to the range of bacteria it can target. A narrow-spectrum antimicrobial specifically targets certain subsets of bacterial pathogens. For instance, some narrow-spectrum drugs only work against gram-positive bacteria, while others only work against gram-negative bacteria. When the pathogen causing an infection is known, it is generally best to use a narrow-spectrum antimicrobial to minimize harm to the normal microbiota.</p>
<p>The bacterial cell wall plays a critical role in defining the spectrum of antimicrobial activity.</p>
<ul>
<li><p><strong>Gram-positive</strong> bacteria have a cell wall consisting of many layers of peptidoglycan totalling 30–100 nm in thickness(<a href="#fig-grampos">Figure&nbsp;<span>4.1</span></a>). These peptidoglycan layers are commonly embedded with teichoic acids (TAs), carbohydrate chains that extend through and beyond the peptidoglycan layer. TA is thought to stabilize peptidoglycan by increasing its rigidity. TA also plays a role in the ability of pathogenic gram-positive bacteria such as Streptococcus to bind to certain proteins on the surface of host cells, enhancing their ability to cause infection.</p>
<div id="fig-grampos" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/gramposwall.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;4.1: Gram positive bacterial cell wall. Source: www.biorender.com</figcaption>
</figure>
</div></li>
<li><p>In addition to peptidoglycan and TAs, bacteria of the family Mycobacteriaceae have an external layer of waxy mycolic acids in their cell wall. These bacteria are referred to as <strong>acid-fast</strong>, since acid-fast stains must be used to penetrate the mycolic acid layer for purposes of microscopy</p></li>
<li><p><strong>Gram-negative</strong> bacteria have a much thinner layer of peptidoglycan (no more than about 4 nm thick) than gram-positive cells, and the overall structure of their cell envelope is more complex (<a href="#fig-gramneg">Figure&nbsp;<span>4.2</span></a>). In gram-negative cells, a gel-like matrix occupies the periplasmic space between the cell wall and the plasma membrane, and there is a second lipid bilayer called the outer membrane, which is external to the peptidoglycan layer. This outer membrane is attached to the peptidoglycan by murein lipoprotein.</p>
<ul>
<li><p>The outer leaflet of the outer membrane contains the molecule lipopolysaccharide (LPS), which functions as an endotoxin in infections involving gram-negative bacteria, contributing to symptoms such as fever, haemorrhaging, and septic shock.</p></li>
<li><p>Each LPS molecule is composed of Lipid A, a core polysaccharide, and an O side chain that is composed of sugar-like molecules that comprise the external face of the LPS (<a href="#fig-gramneg">Figure&nbsp;<span>4.2</span></a>). The composition of the O side chain varies between different species and strains of bacteria. Parts of the O side chain called antigens can be detected using serological or immunological tests to identify specific pathogenic strains like Escherichia coli O157:H7, a deadly strain of bacteria that causes bloody diarrhoea and kidney failure.</p></li>
</ul></li>
</ul>
<div id="fig-gramneg" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/gramnegwall.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.2: Gram negative bacterial cell wall. Source: www.biorender.com</figcaption>
</figure>
</div>
<section id="the-gram-stain-test" class="level3" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="the-gram-stain-test"><span class="header-section-number">4.2.1</span> The Gram-stain test</h3>
<p>The Gram stain procedure is a widely used differential staining technique that was developed by Danish microbiologist Hans Christian Gram in 1884. It enables the differentiation of bacteria based on the characteristics of their cell walls. Even today, it remains one of the most commonly employed staining methods in microbiology and guiding antibiotic therapy. The steps involved in the Gram stain procedure are as follows (<a href="#fig-gramstain">Figure&nbsp;<span>4.3</span></a>):</p>
<div id="fig-gramstain" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/gram.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.3: Gram stain procedure. Source: www.biorender.com</figcaption>
</figure>
</div>
<ol type="1">
<li><p>The first step involves applying crystal violet, a primary stain, to a heat-fixed smear. This primary stain imparts a purple colour to all the cells on the slide.</p></li>
<li><p>Gram’s iodine, an essential component called a mordant, is added as the second step. A mordant acts as a fixing agent, and in this context, Gram’s iodine acts as a trapping agent that forms complexes with the crystal violet. These complexes accumulate in the thick peptidoglycan layers of the cell walls.</p></li>
<li><p>The third step involves adding a decolourising agent, which is typically ethanol or an acetone/ethanol solution. The decolourising agent has a different effect on cells with varying thicknesses of peptidoglycan layers. Those cells with thick peptidoglycan layers are less affected by the decolourising agent, allowing them to retain the purple crystal violet dye and appear purple. In contrast, cells with thinner peptidoglycan layers are more easily decoulorised and lose the crystal violet stain, appearing colourless.</p></li>
<li><p>Finally, a secondary counterstain, usually safranin, is applied. Safranin stains the decolourised cells pink, allowing them to be more easily visualized. However, cells that still contain the retained crystal violet dye are less noticeable since the purple colour masks the pink stain of the counterstain.</p></li>
</ol>
<p>Clinicians use Gram stain results prior to definitive pathogen identification along with knowledge of the most likely pathogens associated with the clinical presentation/differential diagnosis of infection in the patient to select antimicrobial therapy. Spectrum of activity tables such as those compiled in the <a href="https://www.sanfordguide.com/">Sanford Guide</a> and accessible in cell phone applications can be consulted to select an antimicrobial therapy with appropriate spectrum of activity (<a href="#fig-sanford">Figure&nbsp;<span>4.4</span></a>).</p>
<div id="fig-sanford" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/sanford.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.4: Screenshot of an antimicrobial spectrum of activity table from the Sanford Guide to Antimicrobial Therapy cellphone application.</figcaption>
</figure>
</div>
</section>
<section id="why-not-prescribe-the-broadest-spectrum-antimicrobial-for-all-infections" class="level3" data-number="4.2.2">
<h3 data-number="4.2.2" class="anchored" data-anchor-id="why-not-prescribe-the-broadest-spectrum-antimicrobial-for-all-infections"><span class="header-section-number">4.2.2</span> Why not prescribe the broadest-spectrum antimicrobial for all infections?</h3>
<p>Some broad-spectrum antibiotics can target a wide variety of bacterial pathogens, including both gram-positive and gram-negative species. Broad-spectrum antibiotic therapy may be used as <strong>empiric therapy</strong> while waiting for laboratory identification of the infecting pathogen.</p>
<p>Broad-spectrum antimicrobials are frequently used for polymicrobic infections (infections with multiple bacterial species), infectious in immunocompromised patients (where the spectrum of potential pathogens is larger), and for prophylactic prevention of infections for certain surgeries or invasive procedures. In some cases, a broad-spectrum antimicrobial may be chosen to treat an infection when a narrow-spectrum drug fails due to the development of drug resistance by the target pathogen.</p>
<p>However, using broad-spectrum antimicrobials comes with the risk of also targeting a wide range of normal microbiota (<a href="#fig-dysbiosis">Figure&nbsp;<span>4.5</span></a>). <strong>This increases the risk of a superinfection</strong>, which is a secondary infection that occurs in a patient who already has a pre-existing infection. Therefore, we generally want to use the narrowest spectrum of activity appropriate for the type of infection for the shortest period as possible.</p>
<div id="fig-dysbiosis" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/dysbiosis.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.5: Antimicrobial effects on the human microbiota. Source:www.biorender.com</figcaption>
</figure>
</div>
<p>Superinfection can happen when the antibacterial intended for the original infection kills off the protective microbiota, allowing another pathogen that is resistant to the antibacterial to multiply and cause a secondary infection. Examples of superinfection that can develop due to antimicrobial usage include yeast infections (candidiasis), multidrug resistant gram-positive and gram-negative pathogens, and pseudomembranous colitis caused by <em>Clostridium difficile</em>, which can be life-threatening.</p>
</section>
</section>
<section id="bactericidal-versus-bacteriostatic-activity" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="bactericidal-versus-bacteriostatic-activity"><span class="header-section-number">4.3</span> Bactericidal versus bacteriostatic activity</h2>
<p>Antibacterial drugs can have different effects on target bacteria. Some drugs are bacteriostatic, meaning they temporarily inhibit bacterial growth, but the bacteria can resume growth once the drug is eliminated. On the other hand, bactericidal drugs actually kill the target bacteria.</p>
<p>The choice between using bacteriostatic or bactericidal drugs depends on factors such as the type of infection and the patient’s immune status. In patients with strong immune defences, both types of drugs can be effective in achieving a clinical cure. However, for immunocompromised patients, using a bactericidal drug is crucial for successfully treating infections.</p>
<p>In the case of life-threatening infections like acute endocarditis or meningitis, bactericidal antibiotics are generally favoured. However, in recent years the dogma of “bactericidal activity” has been questioned by some infectious diseases specialists as limited data support the superiority of outcomes for regimens defined as bactericidal versus bacteriostatic in them microbiology laboratory.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Is bactericidal activity clinically important?
</div>
</div>
<div class="callout-body-container callout-body">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/spellberg.png" class="img-fluid figure-img"></p>
</figure>
</div>
<p>Is the commonly held belief that bactericidal activity is crucial for certain infections actually challenged by clinical outcome data, as suggested by <a href="spellberg.pdf">this analysis</a> conducted by Wald-Dickler and colleagues?</p>
</div>
</div>
</section>
<section id="antimicrobial-susceptibility-testing--the-minimum-inhibitory-concentration-mic" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="antimicrobial-susceptibility-testing--the-minimum-inhibitory-concentration-mic"><span class="header-section-number">4.4</span> Antimicrobial susceptibility testing- The minimum inhibitory concentration (MIC)</h2>
<p>Antimicrobial susceptibility testing (AST) can be broadly characterized as phenotypic vs.&nbsp;genotypic methods. The tests are further characterized whether the results produce qualitative or quantitative results.</p>
<p>Phenotypic assays detect bacterial cell growth arrest or death in the presence of an antimicrobial therefore allowing true susceptibility testing and can be categorized into either minimal inhibitory con- centration (MIC) or non-MIC methods. Quantitative MIC methods (e.g., broth dilution, agar dilution, and agar gradient diffusion) express results numerically, whereas qualitative non-MIC methods (e.g., disk diffusion) allow for broad categorization by interpretive criteria into susceptible, susceptible-dose dependent, intermediate, non-susceptible or resistant interpretive categories without an MIC value.</p>
<p>Regardless of the method used, traditional phenotypic AST is the final step in the time-consuming and labour-intensive process of clinical microbiology. From specimen collection to Gram stain, organism identification, and AST, the median time is 19 hours, 43 hours, and 65 hours, respectively, for the standard blood culture processing workflow using conventional methods.</p>
<p>Genotypic AST identifies resistance genes via molecular methods and serves as a surrogate for AST, although susceptibility must be confirmed phenotypically. The main advantage of genotypic AST is rapidity in detection of resistance mechanisms that can allow for earlier modification of therapy.</p>
<p><strong>The mean inhibitory concentration (MIC) is a measure of the effectiveness of an antimicrobial agent in inhibiting the growth of microorganisms. It represents the lowest concentration of a specific antimicrobial substance that prevents visible growth of a microorganism after a specified period of incubation-typically 24 hours.</strong></p>
<section id="broth-based-mic-testing-methods." class="level3" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="broth-based-mic-testing-methods."><span class="header-section-number">4.4.1</span> Broth-based MIC testing methods.</h3>
<p>Alexander Fleming developed the first liquid media dilution-based testing method and was a pioneer in the field of antimicrobial susceptibility testing (AST) more than a decade before the penicillin he discovered was available for clinical use. Manual broth microdilution (BMD) is one of the gold standard reference methods per the Clinical and Laboratory Standards Institute (CLSI) and other organizations, such as the U.S. Food and Drug Administration (FDA) and the European Committee on Antimicrobial Susceptibility Testing (EUCAST).</p>
<p>To determine the MIC using broth-based methods, a standardized inoculum of the bacterial is prepared. The inoculum of bacteria is prepared using a turbidity meter instrument against a turbidity standard (typically 0.5 McFarland which represents approximately 1-2 x 10<sup>8</sup> colony forming units (CFU)/mL of inoculum when plated on agar. After dilution, the typical final test inoculum is 5x 10<sup>5</sup> CFU/mL. The intial inoculum can be confirmed by serial dilution if necessary (<a href="#fig-dilution">Figure&nbsp;<span>4.6</span></a>). This standard inoculum is then inoculated into a series of tubes that contain serial dilutions of the antimicrobial agent (<a href="#fig-mictest">Figure&nbsp;<span>4.7</span></a>). After the inoculum is added the tubes are incubated at 35°C for at least 18 hours and inspected for growth (<a href="#fig-mictray2">Figure&nbsp;<span>4.8</span></a>). Growth may also be interpreted visually or by using a spectrophotometer or similar device to detect turbidity or a colour change if an appropriate biochemical substrate that changes colour in the presence of bacterial growth is also included in each well.</p>
<p>The test can be performed in testubes (macrobroth) or microtitre trays (microbroth). The latter is more common because it is simplier to setup and wastes less drug and media.</p>
<div id="fig-dilution" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/dilution.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;4.6: Serial dilution to confirm inoculum. Image source: www.biorender.com</figcaption>
</figure>
</div>
<p>The lowest concentration at which no visible growth is observed is considered the MIC. When samples of clear wells (no visible growth) are plated on antibiotic-free agar, the first clear well that fails to produce growth on agar is considered the mean bactericidal concentration (MBC). This does not mean the MBC culture endpoint is sterile, no organisms, rather the remaining viable inoculum at the MBC endpoint is sufficiently low (i.e.&nbsp;less than 1x 10<sup>2</sup> CFU/mL) such that plating on drug-free agar with a sterile loop (10 μL) will not result in measurable CFUs.</p>
<div id="fig-mictest" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/MICtest.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.7: Procedure for testing the mean inhibitory concentration (MIC) and mean bactericidal concentration (MBC). Source: www.biorender.com</figcaption>
</figure>
</div>
<div id="fig-mictray2" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/MICtray2.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.8: Example of microdilution broth susceptibility testing. In this example, the plate has rows with clindamycin, penicillin, and erythromycin indicated on the left. The drug concentrations increase from left to right. The lowest concentrations that inhibited visible growth for penicillin and erythromycin are indicated by red circles: 0.06 μg/mL for penicillin and 8 μg/mL for erythromycin. For clindamycin, visible bacterial growth was observed at every concentration up to 32 μg/mL, indicating a MIC of &gt;32 μg/mL. Credit CDC and Openstax.</figcaption>
</figure>
</div>
<p>Disk diffusion (Kirby-Bauer) agar-based methods are simpler to perform and can produce reliable MICs, but are not as quantitative and broth-based methods and may not be suitable for some antibiotics that do not diffuse well through the agar.</p>
<p>This method involves placing a disk containing a fixed concentration of antimicrobial onto an agar plate that has been inoculated with standardized bacteria. After incubation, the diameter of the zone of inhibition is measured. While disk diffusion is accurate, it does not provide an MIC value, which is important for dose optimization in certain cases. Therefore, disk zone measurements must be correlated with MIC values obtained through other methods, such as BMD, when establishing and updating clinical breakpoints to minimize errors in categorization (i.e.&nbsp;susceptible, intermediate or resistant).</p>
<div id="fig-agarmic" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/agarmic.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.9: Antimicrobial susceptibility testing on solid media</figcaption>
</figure>
</div>
<p>A method related to the Kirby-Bauer method uses strips imprinted with a gradient of antimicrobials. Several commercial products (Etest, Biomerieux: Liofilchem, Liofilchem diagnsotica) are available. The Etest is an alternative method for determining MIC (<a href="#fig-etest">Figure&nbsp;<span>4.10</span></a>). It combines the Kirby-Bauer disk diffusion test and dilution methods. Instead of circular disks, plastic strips with a gradient of an antibacterial are placed on an agar plate inoculated with bacteria. The drug diffuses into the agar, creating an elliptical zone of inhibition. The intersection of the zone with the drug gradient indicates the MIC. Multiple antimicrobials can be tested simultaneously.</p>
<div id="fig-etest" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/etest.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;4.10: Etest method for susceptibility testing. (Left) design of the gradient strips. (Right) results of six antibiotics. Source: bioMérieux</figcaption>
</figure>
</div>
<p>Agar based methods are less labour-intensive (but non quantitative). They are frequently used for screening susceptibility in urine or other normally sterile fluids.</p>
</section>
</section>
<section id="automated-antimicrobial-susceptibility-testing" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="automated-antimicrobial-susceptibility-testing"><span class="header-section-number">4.5</span> Automated antimicrobial susceptibility testing</h2>
<p>Due to the impracticality of manual BMD for routine AST, the majority of clinical microbiology laboratories use commercial automated conventional phenotypic AST platforms. These platforms employ a miniaturized broth microdilution-based method and automated pathogen identification, AST, and interpretation within a single system.</p>
<p>The most common platforms include:</p>
<ul>
<li><p>MicroScan WalkAway (Beckman Coulter Inc.) <a href="https://www.youtube.com/watch?v=0Lfp6cNM1lU">Youtube video of technology</a>. In this <a href="https://www.youtube.com/watch?v=lz486jTHQ1E">video</a> a microbiology technician shows the steps of how the test is performed.</p></li>
<li><p>PhoenixTM (Becton, Dickinson and Company)<a href="https://www.youtube.com/watch?v=dSK44yZQVLo">Youtube video of technology</a></p></li>
<li><p>SensititreTM (ThermoFisher) <a href="https://www.youtube.com/watch?v=tRtTXbt400w">Youtube video of technology</a></p></li>
<li><p>Vitek2 (bioMérieux) <a href="https://www.youtube.com/watch?v=0DyufjF-P84">Youtube video of the techology</a></p></li>
</ul>
<p>Each of the four systems mentioned can fully automate AST from inoculation to interpretation, although they employ different methodologies that yield varying time to results. The MicroScan, Phoenix, and Sensititre Systems directly measure MICs by observing bacterial growth in the presence of each antimicrobial included in the panel’s wells. The Vitek2 system calculates, rather than measures, MICs by comparing the growth of the test isolate to a reference isolate with a known MIC using a limited range of antimicrobial drug dilutions.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Recommended reading
</div>
</div>
<div class="callout-body-container callout-body">
<p><a href="Wenzler.pdf">Antimicrobial susceptibility testing: An updated primer for clinicians in the era of antimicrobial resistance</a> <span class="citation" data-cites="wenzler">(<a href="references.html#ref-wenzler" role="doc-biblioref">Wenzler et al., n.d.</a>)</span></p>
</div>
</div>
<p><del>Right something here about rapid AST</del></p>
</section>
<section id="rapid-phenotypic-antimicrobial-susceptibility-testing" class="level2" data-number="4.6">
<h2 data-number="4.6" class="anchored" data-anchor-id="rapid-phenotypic-antimicrobial-susceptibility-testing"><span class="header-section-number">4.6</span> Rapid phenotypic antimicrobial susceptibility testing</h2>
<table class="table">
<caption>Rapid antimicrobial susceptibility testing methods. Adapted from Banerjee and Humphries <span class="citation" data-cites="banerjee2021">(<a href="references.html#ref-banerjee2021" role="doc-biblioref">Banerjee and Humphries 2021</a>)</span> {#tbl-rapid} CE-IVD- Certified- in vitro diagnostic European Union.</caption>
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="header">
<th>Test</th>
<th>Technology</th>
<th>Time to result</th>
<th>Regulation status</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Phenotest BC</td>
<td>Time-lapse imaging of bacterial cells under dark-field microscopy. Morphological and kinetic changes analyzed.</td>
<td>7 h</td>
<td>US FDA cleared</td>
</tr>
<tr class="even">
<td>Alfred</td>
<td>Light scattering to detect bacterial growth in liquid culture broth.</td>
<td>3–5 h</td>
<td>CE-IVD</td>
</tr>
<tr class="odd">
<td>dRast</td>
<td>Time-lapse imaging of bacterial cells on micropatterned plastic microchips.</td>
<td>6 h</td>
<td>CE-IVD</td>
</tr>
<tr class="even">
<td>Reveal AST</td>
<td>Sensor array for volatile organic compounds emitted during microorganism growth.</td>
<td>4.5 h</td>
<td>CE-IVD</td>
</tr>
<tr class="odd">
<td>ASTAR (AliFAX)</td>
<td>Time-lapse imaging of bacterial growth in broth</td>
<td>3–6 h</td>
<td></td>
</tr>
<tr class="even">
<td>FastInov</td>
<td>Flow cytometry applying fluorescent dyes that reveal cell damage during treatment.</td>
<td>1.4 h</td>
<td>CE-IVD</td>
</tr>
<tr class="odd">
<td>Lifescale</td>
<td>Mass measurement using a microcantilever.</td>
<td>4 h</td>
<td>CE-IVD</td>
</tr>
</tbody>
</table>
<p><del><em>Write something here</em></del></p>
<ul>
<li><p>T2 bacteria/T2 Candida <a href="https://www.youtube.com/watch?v=HWiX9HNszNM&amp;t=1s">Youtube video on technology</a></p></li>
<li><p>MALDI-TOF Rapid AST <a href="https:/www.youtube.com/watch?v=0jeFpXHZ8W0">Youtube video on technology</a></p></li>
</ul>
<p>Incubation of microorganisms with antimicrobials (or as a growth control without antimicrobials) does not need an additional microtiter plate or another incubation container, because it is performed directly on spots of a MALDI target in form of microdroplets. Another fundamental feature of the method is that read-out is performed by MALDI-TOF MS, in contrast to turbidity reading with reference methods (10, 11). Microorganisms are multiplying without addition of an antimicrobial (growth control), and their amplified bio- mass is detected by MALDI-TOF MS after the incubation. The same holds true for resistant microorganisms that are growing despite the presence of the antimicrobial. In contrast, microorganisms that are susceptible to the tested antimicrobial are inhibited, and their originally inoculated, not-amplified biomass cannot be detected by a MALDI-TOF MS instrument (Fig. 2). The sensitive detection of biomass by MALDI-TOF MS allows incubation times to be considerably shortened in comparison to the reference method.</p>
<section id="genetic-detection-for-rapid-detection-of-resistance-markers" class="level3" data-number="4.6.1">
<h3 data-number="4.6.1" class="anchored" data-anchor-id="genetic-detection-for-rapid-detection-of-resistance-markers"><span class="header-section-number">4.6.1</span> Genetic detection for rapid detection of resistance markers</h3>
<p><del>Writer something here as an introduction</del></p>
<table class="table">
<colgroup>
<col style="width: 16%">
<col style="width: 31%">
<col style="width: 51%">
</colgroup>
<thead>
<tr class="header">
<th>Test</th>
<th>Organisms identified</th>
<th>Resistance genes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Biofire BCID2 (Biofire, Salt Lake City, UT)</td>
<td>9 Gram-positive bacterial targets 14 Gram-negative bacterial targets 7 yeast targets</td>
<td>Carbapenemases blaIMP blaKPC blaOXA−48−like blaNDM blaVIM Colistin resistance mcr-1 ESBL blaCTX−M Methicillin mecA/C MREJ Vancomycin vanA/B</td>
</tr>
<tr class="even">
<td>Verigene BC-GN (Luminex, Austin, TX)</td>
<td>9 Gram-negative bacterial targets</td>
<td>Carbapenemases blaIMP blaKPC blaOXA−48−like blaNDM blaVIM ESBL blaCTX−M</td>
</tr>
<tr class="odd">
<td>Verigene BC-GP (Luminex)</td>
<td>13 Gram-positive bacterial targets</td>
<td>Methicillin mecA MREJ Vancomycin vanA/B</td>
</tr>
<tr class="even">
<td>ePlex® BCID-GP (GenMark, Carlsbad, CA)</td>
<td>20 Gram-positive bacterial targets “pan” Gram-negative target “pan” Candida target</td>
<td>Methicillin mecA MREJ Vancomycin vanA/B</td>
</tr>
<tr class="odd">
<td>ePlex® BCID-GN (GenMark)</td>
<td>21 Gram-negative targets “pan” Gram-positive target “pan” Candida target</td>
<td>Carbapenemases blaIMP blaKPC blaOXA−48/OXA−23 blaNDM blaVIM ESBL blaCTX−M</td>
</tr>
<tr class="even">
<td>ePlex® BCID-FP (GenMark)</td>
<td>15 Fungal targets</td>
<td>None</td>
</tr>
<tr class="odd">
<td>Xpert® MRSA/SA BC (Cepheid, Sunnydale CA)</td>
<td>1 Gram-positive target</td>
<td>Methicillin mecA</td>
</tr>
</tbody>
</table>
<p>Rapid antimicrobial susceptibility testing methods. Adapted from Banerjee and Humphries <span class="citation" data-cites="banerjee2021">(<a href="references.html#ref-banerjee2021" role="doc-biblioref">Banerjee and Humphries 2021</a>)</span> {#tbl-genetic} CE-IVD- Certified- in vitro diagnostic European Union.</p>
</section>
</section>
<section id="technical-limitations-of-phenpotypic-mic-testing" class="level2" data-number="4.7">
<h2 data-number="4.7" class="anchored" data-anchor-id="technical-limitations-of-phenpotypic-mic-testing"><span class="header-section-number">4.7</span> Technical limitations of phenpotypic MIC testing</h2>
</section>
<section id="how-are-mic-results-interpreted" class="level2" data-number="4.8">
<h2 data-number="4.8" class="anchored" data-anchor-id="how-are-mic-results-interpreted"><span class="header-section-number">4.8</span> How are MIC results interpreted?</h2>
</section>
<section id="rapid-genotypic-antimicrobial-susceptibility-testing" class="level2" data-number="4.9">
<h2 data-number="4.9" class="anchored" data-anchor-id="rapid-genotypic-antimicrobial-susceptibility-testing"><span class="header-section-number">4.9</span> Rapid genotypic antimicrobial susceptibility testing</h2>
</section>
<section id="key-concepts-for-review" class="level2" data-number="4.10">
<h2 data-number="4.10" class="anchored" data-anchor-id="key-concepts-for-review"><span class="header-section-number">4.10</span> Key concepts for review</h2>
</section>
<section id="lecture-slides" class="level2" data-number="4.11">
<h2 data-number="4.11" class="anchored" data-anchor-id="lecture-slides"><span class="header-section-number">4.11</span> Lecture slides</h2>
</section>
<section id="references" class="level2" data-number="4.12">
<h2 data-number="4.12" class="anchored" data-anchor-id="references"><span class="header-section-number">4.12</span> References</h2>


<div id="refs" class="references csl-bib-body hanging-indent" role="list" style="display: none">
<div id="ref-banerjee2021" class="csl-entry" role="listitem">
Banerjee, Ritu, and Romney Humphries. 2021. <span>“Rapid Antimicrobial Susceptibility Testing Methods for Blood Cultures and Their Clinical Impact.”</span> <em>Frontiers in Medicine</em> 8: 635831. <a href="https://doi.org/10.3389/fmed.2021.635831">https://doi.org/10.3389/fmed.2021.635831</a>.
</div>
<div id="ref-wenzler" class="csl-entry" role="listitem">
Wenzler, Eric, Mira Maximos, Tomefa E. Asempa, Lauren Biehle, Audrey N. Schuetz, and Elizabeth B. Hirsch. n.d. <span>“Antimicrobial Susceptibility Testing: An Updated Primer for Clinicians in the Era of Antimicrobial Resistance: Insights from the Society of Infectious Diseases Pharmacists.”</span> <em>Pharmacotherapy: The Journal of Human Pharmacology and Drug Therapy</em> n/a (n/a). <a href="https://doi.org/10.1002/phar.2781">https://doi.org/10.1002/phar.2781</a>.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./history.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">History of Antibiotic Development</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./resistance1.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Antibiotic resistance, part 1</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>