<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.361">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>antibioticsbook - 2&nbsp; History of Antibiotic Development</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./antibiotictesting.html" rel="next">
<link href="./intro.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./history.html"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">History of Antibiotic Development</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">antibioticsbook</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./intro.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Multidrug resistance: The ESKAPE pathogens</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./history.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">History of Antibiotic Development</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./antibiotictesting.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Antibiotic testing in the laboratory</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resistance1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Antibiotic resistance, Part 1</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resistance2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Antibiotic resistance, Part 2</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./eskape.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Antibiotic multi</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./discovery1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Antibiotic discovery, Part 1</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./discovery2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Antibiotic discovery, Part 2</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preclinical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Preclinical development of antibiotic candidates</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./toxicities.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Antibiotic toxicities</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./susceptibility.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Clinical application of antimicrobial susceptiblity testing</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./trial.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Clinical trials design</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./economics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Economics of antibiotic discovery</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./diagnostictests.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Antibiotic testing in the laboratory</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./stewardship.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Antimicrobial stewardship</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./lmics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Antibiotic access and use in low-middle income countries</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./mipd.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Model-informed precision dosing and therapeutic drug monitoring</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./summary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Summary</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#the-impact-of-antibiotics-on-human-health" id="toc-the-impact-of-antibiotics-on-human-health" class="nav-link active" data-scroll-target="#the-impact-of-antibiotics-on-human-health">The impact of antibiotics on human health</a>
  <ul class="collapse">
  <li><a href="#ancient-use-of-antibiotics" id="toc-ancient-use-of-antibiotics" class="nav-link" data-scroll-target="#ancient-use-of-antibiotics"><span class="header-section-number">2.0.1</span> Ancient use of antibiotics</a></li>
  <li><a href="#the-modern-era-of-antibiotic-discovery" id="toc-the-modern-era-of-antibiotic-discovery" class="nav-link" data-scroll-target="#the-modern-era-of-antibiotic-discovery"><span class="header-section-number">2.0.2</span> The modern era of antibiotic discovery</a></li>
  <li><a href="#the-discovery-of-penicillin" id="toc-the-discovery-of-penicillin" class="nav-link" data-scroll-target="#the-discovery-of-penicillin"><span class="header-section-number">2.0.3</span> The discovery of penicillin</a></li>
  </ul></li>
  <li><a href="#history-of-antimicrobial-resistance" id="toc-history-of-antimicrobial-resistance" class="nav-link" data-scroll-target="#history-of-antimicrobial-resistance"><span class="header-section-number">2.1</span> History of antimicrobial resistance</a></li>
  <li><a href="#mechanisms-of-antimicrobial-drugs" id="toc-mechanisms-of-antimicrobial-drugs" class="nav-link" data-scroll-target="#mechanisms-of-antimicrobial-drugs"><span class="header-section-number">2.2</span> Mechanisms of antimicrobial drugs</a></li>
  <li><a href="#mechanisms-of-antibiotic-resistance" id="toc-mechanisms-of-antibiotic-resistance" class="nav-link" data-scroll-target="#mechanisms-of-antibiotic-resistance"><span class="header-section-number">2.3</span> Mechanisms of antibiotic resistance</a></li>
  <li><a href="#antibiotic-discovery-to-the-rescue" id="toc-antibiotic-discovery-to-the-rescue" class="nav-link" data-scroll-target="#antibiotic-discovery-to-the-rescue"><span class="header-section-number">2.4</span> Antibiotic discovery to the rescue</a></li>
  <li><a href="#global-response-to-antimicrobial-resistance" id="toc-global-response-to-antimicrobial-resistance" class="nav-link" data-scroll-target="#global-response-to-antimicrobial-resistance">Global response to antimicrobial resistance</a></li>
  <li><a href="#the-global-future-of-amr" id="toc-the-global-future-of-amr" class="nav-link" data-scroll-target="#the-global-future-of-amr">The global future of AMR</a></li>
  <li><a href="#key-concepts-for-review" id="toc-key-concepts-for-review" class="nav-link" data-scroll-target="#key-concepts-for-review"><span class="header-section-number">2.5</span> Key concepts for review</a></li>
  <li><a href="#lecture-slides" id="toc-lecture-slides" class="nav-link" data-scroll-target="#lecture-slides"><span class="header-section-number">2.6</span> Lecture slides</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">History of Antibiotic Development</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Learning objectives
</div>
</div>
<div class="callout-body-container callout-body">
<ul>
<li><p>Compare and contrast natural, semisynthetic, and synthetic antimicrobial drugs</p></li>
<li><p>Describe the historically important individuals and events that led to the development of antimicrobial drugs</p></li>
<li><p>Identify the key challenges in development of future antibiotics</p></li>
</ul>
</div>
</div>
<p><br></p>
<section id="the-impact-of-antibiotics-on-human-health" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="the-impact-of-antibiotics-on-human-health">The impact of antibiotics on human health</h2>
<p>Until the 20th Century, influenza, pneumonia, tuberculosis, and enteric infections ranked among the leading causes of death. The average lifespan for adults in Western Europe was around 50 years, with a 2% mortality rate for children under the age of 5 due to premature deaths primarily caused by infectious diseases.</p>
<p>Industrialization and increasing wealth during the 19th century brought improvements in drinking water and sanitation in many countries, leading to reductions in communicable enteric infections and improved in life expectancy <a href="#fig-lifeexp">Figure&nbsp;<span>2.1</span></a>. By the early 20th century, vaccines for pertussis, diphtheria, yellow fever and tuberculosis were introduced. However, common bacterial infections remained a serious medical threat. Streptococcal throat infections were sometimes fatal, ear infections could progress to deafness, mastoiditis or meningitis. Minor surgeries were associated with risk of life-threatening infections. Maternal mortality during childbirth approached 2%.</p>
<div id="fig-lifeexp" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe src="https://ourworldindata.org/grapher/life-expectancy?tab=chart" loading="lazy" style="width: 100%; height: 600px; border: 0px none;">
</iframe>
<figcaption class="figure-caption">Figure&nbsp;2.1: Changes in life expectancy over 500 years. Source: World in Data</figcaption>
</figure>
</div>
<p>The introduction of effective antibiotic therapy played a crucial role in further reducing mortality rates associated with common infectious diseases. This breakthrough not only improved the safety of childbirth and common surgical procedures but also paved the way for supportive care during chemotherapy for cancer and transplantation. <strong>Thus the discovery of antibiotics can be considered one of the most significant health-related events of modern times.</strong><span class="citation" data-cites="davies_davies10">(<a href="references.html#ref-davies_davies10" role="doc-biblioref">Davies and Davies 2010</a>)</span></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
<section id="ancient-use-of-antibiotics" class="level3" data-number="2.0.1">
<h3 data-number="2.0.1" class="anchored" data-anchor-id="ancient-use-of-antibiotics"><span class="header-section-number">2.0.1</span> Ancient use of antibiotics</h3>
<p>In nature, some microbes, plants and animals have the ability to produce substances that can inhibit or kill microbes that cause disease or compete for the same resources. These natural products have been the source of many antibiotics and treatments with antimicrobial-like activity over the millennia.</p>
<p>The first recorded use of antimicrobial -like substances was by the early Egyptians, Greeks, and Chinese, who used natural products with antimicrobial activity for millennia to treat wounds and infections, even if the causes of these diseases were unknown until the 19th and 20th century (<a href="#fig-ancient">Figure&nbsp;<span>2.2</span></a>). Healers of many cultures understood the antimicrobial properties of fungi and their use of moldy bread or other mold-containing products to treat wounds has been well documented for centuries.<span class="citation" data-cites="wainwright89">(<a href="references.html#ref-wainwright89" role="doc-biblioref">Wainwright 1989</a>)</span> Today, while about 80% of the world’s population still relies on plant-derived medicines,<span class="citation" data-cites="verma_singh08">(<a href="references.html#ref-verma_singh08" role="doc-biblioref">Verma and Singh 2008</a>)</span> scientists are now discovering the active compounds conferring the medicinal benefits contained in many of these traditionally used plants.</p>
<div id="fig-ancient" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ancient.png" class="img-fluid figure-img" width="300"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.2: Recorded use of antimicrobial-like treatments by Chinese (top panel), Egyptians (middle) and Greeks (lower panel). Source: Wikipedia.</figcaption>
</figure>
</div>
<p>However, even today some of the most effective anti-infective treatments (e.g.&nbsp;artemisinin, which are derivatives from “qinghao” or Sweet Wormwood plant) have been “rediscovered” as effective treatments for severe infectious diseases such as malaria (<a href="#fig-chinese">Figure&nbsp;<span>2.3</span></a>).</p>
<p><br></p>
<div id="fig-chinese" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/chinese.png" class="img-fluid figure-img" width="500"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.3: First English-language report of anti-malarial activity of the artemisinins-a traditional Chinese herbal remedy. Artemisinins are now the WHO-recommended front-line treatment for malaria. image course: Chinese Medical Journal</figcaption>
</figure>
</div>
<p><br></p>
<p>Similarly, antibacterial resistance pre-dates the medical use of antibiotics and is estimated to have emerged with bacteria on earth approximately 2-2.5 billion years ago. In contrast, the first humans are believed to have existed around 2 million years ago. Therefore, antibiotic resistance was not created by humans and misuse of antibiotics. <strong>Antibiotic resistance can be considered a biologic certainty and evolutionary destiny.</strong></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
</section>
<section id="the-modern-era-of-antibiotic-discovery" class="level3" data-number="2.0.2">
<h3 data-number="2.0.2" class="anchored" data-anchor-id="the-modern-era-of-antibiotic-discovery"><span class="header-section-number">2.0.2</span> The modern era of antibiotic discovery</h3>
<p>The microbiologist and immunologist Paul Ehrlich (1854-1915) (<a href="#fig-paulerlich">Figure&nbsp;<span>2.4</span></a>) is credited with introduction of <em>strategic antibiotic discovery</em> that with the chemist Sahachiro Hato lead to the discovery and first medical application of the synthetic antibiotic arsphenamine (Salvarsan) for the treatment of a bacterial infection-syphilis. He was awarded the Nobel Prize in Medicine in 1909.</p>
<div id="fig-paulerlich" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Paul_Ehrlich_1915.jpg" class="img-fluid figure-img" width="250"></p>
<figcaption class="figure-caption">Figure&nbsp;2.4: German immunologist Paul Ehrlich (1854-1915). Image: Library of Congress</figcaption>
</figure>
</div>
<p>A few decades later, German scientists Josef Klarer, Fritz Mietzsch, and Gerhard Domagk made a significant discovery regarding the antibacterial properties of a synthetic dye called <strong>prontosil</strong> (<a href="#fig-domagk">Figure&nbsp;<span>2.5</span></a>). They found that it could effectively treat streptococcal and staphylococcal infections in mice. Domagk’s own daughter was one of the first individuals to receive the drug, which successfully cured her severe streptococcal infection caused by a needle jab. Utterly desperate when the doctor recommended amputation to save his daughter’s life, Domagk treated Hildegard with Prontosil, which led to her recovery but she suffered a permanent reddish discoloration of her skin owing to the drug.</p>
<p>In recognition of his work with prontosil and sulfanilamide (the active compound derived from prontosil), Domagk was awarded the Nobel Prize in Medicine in 1939. However, the Nobel committee had angered the German political authorities by awarding the 1935 Nobel Peace Prize to Carl von Ossietzky, an outspoken German pacifist. Under the grip of Hitler and the Nazi Party, German citizens were forbidden to accept the Nobel Prize. After Domagk accepted the prize, he was arrested by the Gestapo and forced to send a letter rejecting it. Although Domagk was able to receive his prize medal in 1947, the prize money had long since been redistributed.</p>
<div id="fig-domagk" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/domagk.png" class="img-fluid figure-img" width="300"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.5: Gerhard Domagk (left) and a vial of prontosil (right), the precursor to sulfanilamide antibiotics</figcaption>
</figure>
</div>
<p>Sulfanilamide, the first synthetic antimicrobial, served as the foundation for the development of a family of sulfa drugs. These successes led to the discovery and production of other important classes of synthetic antimicrobials, such as quinolines and oxazolidinones.</p>
</section>
<section id="the-discovery-of-penicillin" class="level3" data-number="2.0.3">
<h3 data-number="2.0.3" class="anchored" data-anchor-id="the-discovery-of-penicillin"><span class="header-section-number">2.0.3</span> The discovery of penicillin</h3>
<p>A few years prior to the prontosil discovery, Alexander Fleming made an accidental but ground breaking finding. In 1928, while examining old plates of staphylococci in his laboratory, Fleming noticed that mold growth (later identified as <em>Penicillium notatum</em>) inhibited the growth of staphylococci on one plate (<a href="#fig-pcndisc">Figure&nbsp;<span>2.6</span></a>). This marked the discovery of penicillin, the first natural antibiotic. Further experiments carried out by Chain and Florey demonstrated penicillin’s antibacterial properties against streptococci, meningococci, and <em>Corynebacterium diphtheriae</em>, the causative agent of diphtheria.</p>
<p>While Fleming and his colleagues were credited with the discovery and identification of penicillin, its isolation and mass production were achieved by a team of researchers at Oxford University led by Howard Florey and Ernst Chain. In 1940, they purified penicillin and reported its success as an antimicrobial agent against streptococcal infections in mice. Subsequent studies with human subjects also proved penicillin’s effectiveness. For their significant contributions, Fleming, Florey, and Chain were awarded the Nobel Prize in Physiology and Medicine in 1945.</p>
<div id="fig-pcndisc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/pcndisc.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.6: Original report of penicillin antimicrobial activity in the British Journal of Pathology 1928. The discovers and developers of penicillin.</figcaption>
</figure>
</div>
<p>In June of 1943 Mary Hunt, a lab assistant working in Peoria, Illinois, found a cantaloupe at a local market covered in mold with a “pretty, golden look.” (<a href="#fig-moldymarry">Figure&nbsp;<span>2.7</span></a>). This mold turned out to be a highly productive strain of <em>Penicillium chrysogeum</em> and its discovery marked a turning point in the quest to mass produce penicillin.</p>
<div id="fig-moldymarry" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/moldymary.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.7: Penicillin fermentation (left panel), Douglas Gorsline’s 1948 painting of “Moldy Mary” in Peoria Illinois(center panel), Purported original photograph of the moldy mellon (right panel)</figcaption>
</figure>
</div>
<div class="callout callout-style-simple callout-none no-icon">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-body-container">
<p>For further reading about the history of penicillin development, see the excellent book by Eric Lax: <em>The Mold in Dr.&nbsp;Florey’s Coat</em></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/lax.png" class="img-fluid figure-img" width="150"></p>
</figure>
</div>
</div>
</div>
</div>
<p>In the early 1940s, Dorothy Hodgkin, a crystallography expert at Oxford University, used X-rays to analyze the structure of various natural products. In 1946, she determined the structure of penicillin, earning her the Nobel Prize in Chemistry in 1964 (<a href="#fig-hodgkin">Figure&nbsp;<span>2.8</span></a>). Knowledge of the penicillin structure allowed scientists to modify penicillin, leading to the development of semisynthetic versions. Semisynthetic antimicrobials are chemically modified derivatives of natural antibiotics, designed to broaden their bacterial targets, increase stability, reduce toxicity, or confer other beneficial properties for treating infections.</p>
<div id="fig-hodgkin" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/hodgkin.png" class="img-fluid figure-img" width="150"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.8: Dorothy Hodgkin is the biochemist who solved the xray crystal structure of penicillin opening the door for synthesis of semisynthetic penicillins to combat resistance. Image source: www.nobelprize.org</figcaption>
</figure>
</div>
<p>Penicillin is just one example of a natural antibiotic. In the 1940s, Selman Waksman, a renowned soil microbiologist at Rutgers University, and his research team discovered several antimicrobials, including actinomycin, streptomycin, and neomycin. These findings stemmed from Waksman’s study of fungi and Actinobacteria, particularly soil bacteria in the <em>Streptomyces</em> genus known for their natural production of diverse antimicrobials (<a href="#fig-waksman">Figure&nbsp;<span>2.9</span></a>).</p>
<div id="fig-waksman" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/waksman.png" class="img-fluid figure-img" width="300"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.9: Selman Wakesman. Winner, Nobel Prize in Medicine in 1952. Source: Rutgers University</figcaption>
</figure>
</div>
<p>Waksman’s contributions earned him the Nobel Prize in Physiology and Medicine in 1952. Actinomycetes, the source of more than half of all natural antibiotics, continue to serve as a valuable resource for discovering novel antimicrobial agents. Some researchers believe that we have yet to fully explore the antimicrobial potential of this group.</p>
<p>The term <em>antibiotic</em> was actually coined by Waksman.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
</section>
</section>
<section id="history-of-antimicrobial-resistance" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="history-of-antimicrobial-resistance"><span class="header-section-number">2.1</span> History of antimicrobial resistance</h2>
<p>Antimicrobial resistance is a natural occurrence as microbes adapt to overcome antimicrobial compounds produced by other microorganisms. However, the development and widespread use of antimicrobial drugs by humans have added another selective pressure that drives further evolution. Factors such as overuse, misuse, inappropriate use, subtherapeutic dosing, and patient non-compliance and widespread use of antibiotics in food production (livestock can accelerate the evolution of drug resistance. Pathogens can develop resistance through chromosomal mutations that are vertically transferred to subsequent generations or through horizontal gene transfer facilitated by plasmids and transposons, which can promote the spread of resistance.</p>
<p>Certain organisms, like <em>Acinetobacter baumannii</em> or <em>Pseudomonas aeruginosa</em>, naturally possess the ability to develop multiple types of resistance. On the other hand, organisms like <em>Klebsiella pneumoniae</em> have historically been treatable but are now becoming highly resistant by acquiring new resistance elements. There are also organisms, such as <em>Streptococcus pyogenes</em>, that have remained susceptible to “old” antibiotics like penicillin since their introduction.</p>
<p>Antibiotic resistance in bacteria typically originates from specific events. While resistance can arise within a bacterium through random mutations affecting the target of the antibiotic or other essential elements, it is more commonly acquired from other bacteria through the transfer of mobile genetic elements. Bacteria have the ability to exchange genes not only within their own species but also between different species and even genera. The transfer of <strong>plasmids</strong>, circular DNA structures that can carry multiple genes including those for antibiotic resistance, is a crucial mechanism for gene transmission among bacteria. Plasmids are highly mobile and can encode for various types of resistance that are unrelated, such as resistance to cephalosporins through beta-lactamase production and resistance to fluoroquinolones via efflux pumps. Through this gene swapping process, a bacterium can become resistant to multiple antibiotics.</p>
<p>Penicillinase (enzymes produced by bacteria that destroy the beta-lactam ring of penicillin) were already reported by 1940 prior to the first clinical use of the drug.<span class="citation" data-cites="davies_davies10">(<a href="references.html#ref-davies_davies10" role="doc-biblioref">Davies and Davies 2010</a>)</span> Alexander Fleming was among the first physicians to caution about the risks of resistance to penicillin if used too little or for a too short of period during treatment.</p>
<p><br></p>
<div class="callout callout-style-simple callout-none no-icon">
<div class="callout-body d-flex">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-body-container">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/alexflemming.jpeg" class="img-fluid figure-img" width="150"></p>
</figure>
</div>
<p>“It is not difficult to make microbes resistant to penicillin in the laboratory by exposing them to concentrations not sufficient to kill them, and the same thing has occasionally happened in the body. The time may come when penicillin can be bought by anyone in the shops. Then there is the danger that the ignorant man may easily under-dose himself and by exposing his microbes to non-lethal quantities of the drug make them resistant.” <br> <br> Sir Alexander Fleming, Nobel Prize Lecture, December 11, 1944</p>
</div>
</div>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
</section>
<section id="mechanisms-of-antimicrobial-drugs" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="mechanisms-of-antimicrobial-drugs"><span class="header-section-number">2.2</span> Mechanisms of antimicrobial drugs</h2>
<p>Selective toxicity is a crucial trait for antimicrobial drugs, as it means they can specifically target and harm microbial organisms while causing minimal harm to the host. Antibacterial drugs are the most commonly used antimicrobials because they have a wider range of unique targets in prokaryotic cells compared to fungi, parasites, and viruses. Each class of antibacterial drugs has its own distinct mode of action, which describes how the drug affects microbes at the cellular level. See <a href="#fig-bactmechanism">Figure&nbsp;<span>2.10</span></a> and <a href="#tbl-mechanism">Table&nbsp;<span>2.1</span></a>).</p>
<div id="fig-bactmechanism" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/bactmechanism.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2.10: Mechanisms of antibacterial agents. Image source: OpenStax-Rice University.</figcaption>
</figure>
</div>
<p><br> <br></p>
<div id="tbl-mechanism" class="anchored">
<table class="table">
<caption>Table&nbsp;2.1: Mechanism of antimicrobial agents</caption>
<colgroup>
<col style="width: 33%">
<col style="width: 31%">
<col style="width: 34%">
</colgroup>
<thead>
<tr class="header">
<th><strong>Common Antibacterial Drugs by Mode of Action</strong></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Mode of Action</td>
<td>Target</td>
<td>Drug Class</td>
</tr>
<tr class="even">
<td>Inhibit cell wall biosynthesis</td>
<td>Penicillin-binding proteins</td>
<td>β-lactams: penicillins, cephalosporins, monobactams, carbapenems</td>
</tr>
<tr class="odd">
<td>Peptidoglycan subunits</td>
<td>Glycopeptides</td>
<td></td>
</tr>
<tr class="even">
<td>Peptidoglycan subunit transport</td>
<td>Bacitracin</td>
<td></td>
</tr>
<tr class="odd">
<td>Inhibit biosynthesis of proteins</td>
<td>30S ribosomal subunit</td>
<td>Aminoglycosides, tetracyclines</td>
</tr>
<tr class="even">
<td>50S ribosomal subunit</td>
<td>Macrolides, lincosamides, chloramphenicol, oxazolidinones</td>
<td></td>
</tr>
<tr class="odd">
<td>Disrupt membranes</td>
<td>Lipopolysaccharide, inner and outer membranes</td>
<td>Polymyxin B, colistin, daptomycin</td>
</tr>
<tr class="even">
<td>Inhibit nucleic acid synthesis</td>
<td>RNA</td>
<td>Rifamycin</td>
</tr>
<tr class="odd">
<td>DNA</td>
<td>Fluoroquinolones</td>
<td></td>
</tr>
<tr class="even">
<td>Antimetabolites</td>
<td>Folic acid synthesis enzyme</td>
<td>Sulfonamides, trimethoprim</td>
</tr>
<tr class="odd">
<td>Mycolic acid synthesis enzyme</td>
<td>Isonicotinic acid hydrazide</td>
<td></td>
</tr>
<tr class="even">
<td>Mycobacterial adenosine triphosphate (ATP) synthase inhibitor</td>
<td>Mycobacterial ATP synthase</td>
<td>Diarylquinoline</td>
</tr>
</tbody>
</table>
</div>
<p><br></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
</section>
<section id="mechanisms-of-antibiotic-resistance" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="mechanisms-of-antibiotic-resistance"><span class="header-section-number">2.3</span> Mechanisms of antibiotic resistance</h2>
<p>There are several common mechanisms for drug resistance, which are summarized in <a href="#fig-resistance">Figure&nbsp;<span>2.11</span></a>. These mechanisms include enzymatic modification of the drug, modification of the antimicrobial target, and prevention of drug penetration or accumulation.</p>
<div id="fig-resistance" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/resistance.png" class="img-fluid figure-img"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.11: Overview of antimicrobial resistance mechanisms</figcaption>
</figure>
</div>
<p><strong>Drug Modification or Inactivation:</strong> Resistance genes can produce enzymes that modify or hydrolyse antimicrobials, rendering them inactive. This mechanism is common in various antimicrobials, such as aminoglycosides and β-lactams. Aminoglycoside resistance can occur through the transfer of chemical groups to the drug molecule, interfering with its binding. β-lactam resistance involves the enzymatic hydrolysis of the β-lactam bond in the drug molecule’s β-lactam ring, leading to loss of antibacterial activity. Rifampin can be inactivated through glycosylation, phosphorylation, or ADP ribosylation, while macrolides and lincosamides can be enzymatically inactivated or modified.</p>
<p><strong>Prevention of Cellular Uptake or Efflux:</strong> Some microbes develop resistance by hindering the accumulation of antimicrobial drugs, preventing them from reaching their targets. Gram-negative pathogens can alter outer membrane lipid composition, porin channel selectivity, or porin channel concentrations. For instance, <em>Pseudomonas aeruginosa</em> reduces the amount of its OprD porin to resist carbapenems. Efflux pumps are produced by many gram-positive and gram-negative bacteria to actively transport drugs out of the cell, preventing effective drug accumulation. Efflux pumps can confer resistance to β-lactams, tetracyclines, and fluoroquinolones.</p>
<p><strong>Target Modification:</strong> Structural changes in antimicrobial drug targets can prevent drug binding, leading to resistance. Spontaneous mutations in genes encoding antibacterial drug targets allow bacteria to develop resistance. For example, changes in penicillin-binding proteins (PBPs) can inhibit the binding of β-lactam drugs, providing resistance. Streptococcus pneumoniae alters its own PBPs, while <em>Staphylococcus aureus</em> acquires low-affinity PBPs to resist methicillin. Other examples include alterations in ribosome subunits, lipopolysaccharide structure, RNA polymerase, DNA gyrase, metabolic enzymes, and peptidoglycan subunit peptide chains.</p>
<p><strong>Target Overproduction or Enzymatic Bypass:</strong> In antimetabolite drugs that target specific enzymes, resistance can occur through target overproduction or the development of bypass mechanisms. Microbes may produce excess target enzymes, ensuring enough enzyme activity despite the presence of the drug. Alternatively, they may develop bypass pathways that do not rely on the functional target enzyme. Sulfonamide resistance can occur through these mechanisms. Vancomycin resistance in <em>S. aureus</em> involves decreased cross-linkage of peptide chains in the cell wall, providing more targets for vancomycin binding.</p>
<p><strong>Target Mimicry:</strong>A recently discovered resistance mechanism called target mimicry involves the production of proteins that prevent drugs from binding to their targets. For example, <em>Mycobacterium tuberculosis</em> can develop fluoroquinolone resistance by producing a protein called MfpA, which resembles DNA. MfpA binds to DNA gyrase, preventing fluoroquinolones from binding.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/breakblue.png" class="img-fluid figure-img" width="100"></p>
</figure>
</div>
</section>
<section id="antibiotic-discovery-to-the-rescue" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="antibiotic-discovery-to-the-rescue"><span class="header-section-number">2.4</span> Antibiotic discovery to the rescue</h2>
<p>Initially, antibiotic discovery seemed to keep pace with emerging resistance as a host of new chemical classes were developed and introduced in the 1950s-1980’s. For the first half of the century, the repeated and successful response to emerging resistance was to discover a new class of antibiotics.</p>
<p>Yet by the 1980’s, the discovery of new agents began to slow and this strategy began to fail <a href="#fig-abxtimeline">Figure&nbsp;<span>2.12</span></a>. The last truly “new” antibiotic class discovered that reached the market was in 1987. Since then, there has been a lack of innovation in the field, and today there are few novel antibiotic classes in the drug pipeline.</p>
<p><br></p>
<div id="fig-abxtimeline" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/ab-discovery-timeline.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2.12: Antibiotic discovery timeline. Source: www.reactgroup.org</figcaption>
</figure>
</div>
<p>The slowdown in new antibiotic development is occurring at a time when resistance, particularly to penicillins and beta-lactam antibiotics is dramatically increasing due to the wide spread and adaptation of enzymes that hydrolyze this drug class-beta lactamases (<a href="#fig-betalactmases">Figure&nbsp;<span>2.13</span></a>).</p>
<div id="fig-betalactmases" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/betalactamases.png" class="img-fluid figure-img" width="1500"></p>
</figure>
</div>
<figcaption class="figure-caption">Figure&nbsp;2.13: Marked increased in beta-lactamases over the last 4 decades. Figure is form Bush et al. <span class="citation" data-cites="bush2010">(<a href="references.html#ref-bush2010" role="doc-biblioref">Bush and Jacoby 2010</a>)</span></figcaption>
</figure>
</div>
<p>Antibiotic use in humans in only one of many drivers of antimicrobial resistance, as increasing use of antimicrobials in livestock for growth promotion, of agriculture to prevent spoilage, or release of antimicrobials into the environmental waste-water contributes to the <em>resistome</em> - the total burden of antibiotic resistance genes in the environment (<a href="#fig-resistancedrivers">Figure&nbsp;<span>2.14</span></a>}. Therefore, strategies to combat antimicrobial resistance have taken on a <strong>One Health Approach- i.e.&nbsp;interdisciplinary and collaborative strategies that considers the interconnectedness of human health, animal health, and the environment. This approach promotes cooperation between various sectors, such as human medicine, veterinary medicine, environmental science, and public health, to achieve optimal health outcomes for all.</strong></p>
<p><br></p>
<div id="fig-resistancedrivers" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/modifiablerisk.png" class="img-fluid figure-img" width="500"></p>
<figcaption class="figure-caption">Figure&nbsp;2.14: Modifiable risk factors that drive antimicrobial resistance. Figure form Homes et al.<span class="citation" data-cites="holmes2016">(<a href="references.html#ref-holmes2016" role="doc-biblioref">Holmes et al. 2016</a>)</span></figcaption>
</figure>
</div>
<p><strong>The consequences of faltering antibiotic discovery are now being felt worldwide as more and more bacterial infections are becoming harder to treat.</strong> Especially worrisome is the lack of antibiotics against common Gram-negative bacteria (i.e.&nbsp;<em>Escherichia coli, Klebsiella pneumonia, Pseudomonas aeruginosa, Acinetobacter baumannii</em>) that are increasingly resistant to all but last-line antibiotics. The rapid global spread of multi- and pan-resistant bacteria, also known in the lay press as “superbugs,” can cause infections that are not treatable with existing antibiotics.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
How quickly can antibiotics develop resistance?
</div>
</div>
<div class="callout-body-container callout-body">
<p>Click this link to watch a YOUTUBE video of how quickly <em>Escherichia coli</em> and <a href="https://www.youtube.com/watch?v=bDa4-nSc7J8">develop resistance to ciprofloxacin.</a></p>
<p>Another excellent YOUTUBE video on the crises of antibiotic resistance and the dwindleing antibiotic pipeline was presented by the FRONTLINE Program on the : <a href="https://www.youtube.com/watch?v=EkyAuG9RSSU&amp;t=1250s">When Antibiotics Don’t Work</a></p>
</div>
</div>
</section>
<section id="global-response-to-antimicrobial-resistance" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="global-response-to-antimicrobial-resistance">Global response to antimicrobial resistance</h2>
<p>Recognizing the growing global threat of antibiotic resistance (AMR) on human health but also the economy and human development, The World Health Organization (WHO) and The Organisation for Animal Health (OIE) developed in 2001 a Global Plan for Containment of Antimicrobial Resistance that subsequently led to a <a href="https://www.who.int/publications/i/item/9789241509763">Global Action Plan for AMR</a> in 2017 and more recently a <a href="https://www.who.int/news/item/30-07-2021-call-to-action-on-antimicrobial-resistance-2021">“Call to Action on Antimicrobial Resistance”</a> in 2021. The plan outlines 21 strategies and 5 strategic objectives action plans that should be implemented in member states to address AMR. These include:</p>
<ol type="1">
<li>Improvements in the awareness and understanding of antimicrobial resistance through effective communication, education and training</li>
<li>Strengthening of knowledge and evidence base of AMR through surveillance and research</li>
<li>Reductions in the incidence of infection through effective sanitation, hygiene and infection prevention measures</li>
<li>Optimization the use of antimicrobial medicines in human and animal health</li>
<li>Development of an economic case for sustainable investment in AMR research that takes account of the needs of all countries, and increase investment in new medicines, diagnostic tools, vaccines and other interventions</li>
</ol>
<p>One aspect of this action plan was the development of Global Antimicrobial Resistance and Use Surveillance System (GLASS). A link the 2022 report can be <a href="glass2022.pdf">here</a></p>
<p>The WHO also proposed a <a href="https://www.who.int/medicines/publications/WHO-PPL-Short_Summary_25Feb-ET_NM_WHO.pdf">Priority Pathogen List</a> for research and development of new antibiotics and established a global antimicrobial use and surveillance program. This list includes bacterial pathogens that are considered to be be the biggest threat to human health in addition to <em>Mycobacterium tuberculosis.</em> The WHO list breaks down pathogens into three groups:</p>
<div id="tbl-whopriority" class="anchored">
<table class="table">
<caption>Table&nbsp;2.2: WHO priority pathogens</caption>
<colgroup>
<col style="width: 13%">
<col style="width: 86%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Priority</th>
<th style="text-align: left;">Pathogens included</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Critical</strong></td>
<td style="text-align: left;"><p><em>Acinetobacter baumannii</em> (Carbapenem-resistant)</p>
<p><em>Pseudomonas aeruginosa</em> (Carbapenem-resistant)</p>
<p>Enterbacterales (3rd generation cephalosporin, carbapenem-resistant)</p></td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>High</strong></td>
<td style="text-align: left;"><p><em>Enterococcus faecium</em>, vancomycin-resistant</p>
<p><em>Staphylococcus aureus</em>, methicillin-resistant, vancomycin intermediate and resistant</p>
<p><em>Helicobacter pylori</em>, clarithromycin-resistant</p>
<p><em>Campylobacter</em>, fluoroquinolone-resistant</p>
<p><em>Salmonella</em> spp., fluoroquinolone-resistant</p>
<p><em>Neisseria gonorrhoeae</em>, 3rd generation cephalosporin-resistant, fluoroquinolone-resistant</p></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Medium</strong></td>
<td style="text-align: left;"><p><em>Streptococcus pneumoniae</em>, penicillin-non-susceptible</p>
<p><em>Haemophilus influenzae</em>, ampicillin-resistant</p>
<p><em>Shigella</em> spp., fluoroquinolone-resistant</p></td>
</tr>
</tbody>
</table>
</div>
<p>These pathogens may exhibit multi-drug resistance (MDR), extensive drug resistance (XDR) or pan-drug resistance (PDR).<span class="citation" data-cites="magiorakos2012">(<a href="references.html#ref-magiorakos2012" role="doc-biblioref">Magiorakos et al. 2012</a>)</span> Difficult-to-treat resistance (DTR) is a newer definition used to define isolate resistance patterns that require the use of less-effective or more toxic “reserve” antibiotics- e.g., <em>Acinetobacter baumannii</em> susceptible only to colistin and tobramycin.<span class="citation" data-cites="kadri2018">(<a href="references.html#ref-kadri2018" role="doc-biblioref">Kadri et al. 2018</a>)</span></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
What do these resistance definitions mean?
</div>
</div>
<div class="callout-body-container callout-body">
<p>The resistance definitions used by the WHO have specific meanings.</p>
<ul>
<li>MDR-resistance to one agent in at least 3 antibiotic categories</li>
<li>XDR-resistant except to 2 or fewer antibiotic categories</li>
<li>PDR-resistant to all agents in all antibiotic categories</li>
<li>DTR-requires the use of less-effective or more toxic “reserve” antibiotics</li>
</ul>
</div>
</div>
<p>Currently, both the <a href="https://apps.who.int/iris/bitstream/handle/10665/312266/9789241515528-eng.pdf">WHO</a> and <a href="https://www.oie.int/en/what-we-do/global-initiatives/antimicrobial-resistance/#ui-id-4">OIE</a> have also developed lists of antibiotics that are considered of “critical importance” for human and animal medicine. These lists help establish priorities for antimicrobial resistance surveillance and new drug development.</p>
</section>
<section id="the-global-future-of-amr" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="the-global-future-of-amr">The global future of AMR</h2>
<ul>
<li>Drug-resistant infections already cause at least 700,000 deaths globally a year, including 230,000 deaths from multidrug-resistant tuberculosis.</li>
<li><strong>The estimated total number of deaths due to AMR could climb to 10 million deaths globally per year by 2050 under current projections</strong> <a href="#fig-oneilreport">Figure&nbsp;<span>2.15</span></a>.</li>
<li>Increasing resistance could lead to an unthinkable future of untreatable infections, reversing more than a 100 years of medical progress.
<ul>
<li><p>Routine medical procedures or surgery will become more dangerous and associated with higher complication rates.</p></li>
<li><p>Immunosuppression, cancer chemotherapy and transplantation may carry unacceptable risk for many patients if infections cannot be effectively prevented and treated.</p></li>
</ul></li>
<li>Economic and social progress in many countries will be dramatically impacted by increasing AMR leading to political and social instability. The initial short-term economic damage of uncontrolled antimicrobial resistance will be comparable to the economic shocks experienced during the 2008-2009 global financial crisis and result in dramatically-increased healthcare expenditures; reductions in food and feed production, reduced economic output, and <a href="https://documents.worldbank.org/en/publication/documents-reports/documentdetail/323311493396993758/final-report">increased poverty and inequality</a>. The economic impact of antimicrobial resistance is predicted to be even greater and longer lasting on low-and middle-income (LMIC) countries.</li>
</ul>
<div id="fig-oneilreport" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/AMR_deaths_2050.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2.15: Projected deaths due to antimicrobial resistance in 2050. Source: O’Neil Report.</figcaption>
</figure>
</div>
</section>
<section id="key-concepts-for-review" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="key-concepts-for-review"><span class="header-section-number">2.5</span> Key concepts for review</h2>
<ul>
<li>Concept 1</li>
<li>Concept 2</li>
<li>Concept 3</li>
</ul>
</section>
<section id="lecture-slides" class="level2" data-number="2.6">
<h2 data-number="2.6" class="anchored" data-anchor-id="lecture-slides"><span class="header-section-number">2.6</span> Lecture slides</h2>


<div id="refs" class="references csl-bib-body hanging-indent" role="list" style="display: none">
<div id="ref-bush2010" class="csl-entry" role="listitem">
Bush, Karen, and George A. Jacoby. 2010. <span>“Updated Functional Classification of β-Lactamases.”</span> <em>Antimicrobial Agents and Chemotherapy</em> 54 (3): 969–76. <a href="https://doi.org/10.1128/AAC.01009-09">https://doi.org/10.1128/AAC.01009-09</a>.
</div>
<div id="ref-davies_davies10" class="csl-entry" role="listitem">
Davies, Julian, and Dorothy Davies. 2010. <span>“Origins and <span>Evolution</span> of <span>Antibiotic Resistance</span>.”</span> <em>Microbiology and Molecular Biology Reviews : MMBR</em> 74 (3): 417–33. <a href="https://doi.org/10.1128/MMBR.00016-10">https://doi.org/10.1128/MMBR.00016-10</a>.
</div>
<div id="ref-holmes2016" class="csl-entry" role="listitem">
Holmes, Alison H, Luke S P Moore, Arnfinn Sundsfjord, Martin Steinbakk, Sadie Regmi, Abhilasha Karkey, Philippe J Guerin, and Laura J V Piddock. 2016. <span>“Understanding the Mechanisms and Drivers of Antimicrobial Resistance.”</span> <em>The Lancet</em> 387 (10014): 176–87. <a href="https://doi.org/10.1016/S0140-6736(15)00473-0">https://doi.org/10.1016/S0140-6736(15)00473-0</a>.
</div>
<div id="ref-kadri2018" class="csl-entry" role="listitem">
Kadri, Sameer S, Jennifer Adjemian, Yi Ling Lai, Alicen B Spaulding, Emily Ricotta, D Rebecca Prevots, Tara N Palmore, et al. 2018. <span>“Difficult-to-Treat Resistance in Gram-Negative Bacteremia at 173 US Hospitals: Retrospective Cohort Analysis of Prevalence, Predictors, and Outcome of Resistance to All First-Line Agents.”</span> <em>Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America</em> 67 (12): 18031814. <a href="https://doi.org/10.1093/cid/ciy378">https://doi.org/10.1093/cid/ciy378</a>.
</div>
<div id="ref-magiorakos2012" class="csl-entry" role="listitem">
Magiorakos, A-P, A Srinivasan, R B Carey, Y Carmeli, M E Falagas, C G Giske, S Harbarth, et al. 2012. <span>“Multidrug-Resistant, Extensively Drug-Resistant and Pandrug-Resistant Bacteria: An International Expert Proposal for Interim Standard Definitions for Acquired Resistance.”</span> <em>Clinical Microbiology and Infection: The Official Publication of the European Society of Clinical Microbiology and Infectious Diseases</em> 18 (3): 268281. <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-0691.2011.03570.x/pdf">http://onlinelibrary.wiley.com/doi/10.1111/j.1469-0691.2011.03570.x/pdf</a>.
</div>
<div id="ref-verma_singh08" class="csl-entry" role="listitem">
Verma, Sheetal, and S. P. Singh. 2008. <span>“Current and Future Status of Herbal Medicines.”</span> <em>Veterinary World</em> 1 (11): 347.
</div>
<div id="ref-wainwright89" class="csl-entry" role="listitem">
Wainwright, Milton. 1989. <span>“Moulds in Ancient and More Recent Medicine.”</span> <em>Mycologist</em> 3 (1): 21–23.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./intro.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Multidrug resistance: The ESKAPE pathogens</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./antibiotictesting.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Antibiotic testing in the laboratory</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>